docs/manage/get_support.rst

```diff
@@ -56,16 +56,38 @@ The ``sqlite3`` command is necessary. This can be installed with ``sudo apt inst
       sqlite3 db.sqlite3 .dump | sqlite3 fixed.db
       mv fixed.db db.sqlite3
 
-#. Remove the following files.
+#. Remove temporary database files.
    
     .. code-block:: bash
 
-      rm -f db.sqlite3-wal db.sqlite3-shm job_storage.sqlite3 /process_cache/cache.db
+      rm -f db.sqlite3-* job_storage.sqlite3* notifications.sqlite3* process_cache/cache.db
```

Comment:
Several changes:

* Note the forward slash `/` removed prepending the `process_cache` got removed.
* I also added `notifications.sqlite3`
* All paths got a little asterisk `*` so to remove both `-wal` and `-shm` which weren't restricted to just  `db.sqlite3`